style/app.scss

```diff
@@ -687,131 +642,121 @@ a:active
-#sidebar-icon
+.expanded > .sidebar-button > #sidebar-icon
 {
-  width        : 30px;
-  height       : 50px;
-  padding-top  : 20px;
-  padding-left : 2px;
-  position     : absolute;
-  top          : 40%;
-  left         : 4px;
+  -webkit-transform : rotate(180deg);
+  -moz-transform    : rotate(180deg);
+  -ms-transform     : rotate(180deg);
+  -o-transform      : rotate(180deg);
+  transform         : rotate(180deg);
 }
 
-#focuses-nav, #graph-nav
+div#sidebar.collapsed
 {
-  cursor : pointer;
+  height: 40px;
 }
 
-#sidebar-nav
+.collapsed > .sidebar-dropdown
 {
-  width            : 100%;
-  font-size        : 13px;
-  background-color : #cac4d4;
-  border           : solid 1px #dedede;
-  box-shadow       : 0 2px 2px -1px rgba(0, 0, 0, 0.055);
-  display          : block;
-  overflow         : hidden;
+  display: none;
+  visibility: hidden;
 }
 
-#sidebar-nav ul
+.sidebar-button
 {
-  width  : 100%;
-  margin : 0 0 0 0;
+  cursor           : pointer;
+  border-radius    : 10px;
+  width            : 30px;
+  height           : 15px;
+  background-color : #d9d9d9;
+  margin-top       : -5px;
+  margin-left      : 60px;
+  text-align       : center;
 }
 
-#sidebar-nav ul li
+.selected-courses
 {
-  list-style-type    : none;
-  display            : inline-block;
-  width              : 48%;
-  -webkit-transition : all 0.2s;
-  -moz-transition    : all 0.2s;
-  -ms-transition     : all 0.2s;
-  -o-transition      : all 0.2s;
-  transition         : all 0.2s;
+  margin-top       : 0px;
+  margin-bottom    : 10px;
+  font-size        : 12px;
+  color            : #a47cb8;
 }
 
-#sidebar-nav ul li:hover
+.courses
 {
-  background-color : #5C497E !important;
+  overflow         : auto;
+  max-height       : 275px;
+  max-width        : 120px;
+  display          : inline-block;
 }
 
-#sidebar-nav ul li:hover a
+.course-selection
 {
-  color : white !important;
+  font-size  : 12px;
 }
 
-#sidebar-nav ul li .nav-section
+.sidebar-button:hover
 {
-  color           : #000000;
-  display         : inline-block;
-  line-height     : 30px;
-  text-align      : center;
-  width           : 95%;
-  text-decoration : none;
+  background-color: #cac4d4;
 }
 
-#focuses, #graphs
+#sidebar-icon
 {
-  margin-top  : 25px;
-  margin-left : 25px;
-  height      : 86%;
-  overflow-y  : scroll;
+  width : 15px;
+  height: 15px;
 }
 
-.focus
+.fcecount
 {
-  display          : block;
-  cursor           : pointer;
-  font-size        : 20px;
-  border           : solid 1px #000000;
-  text-align       : center;
-  width            : 90%;
+  border-radius    : 10px;
+  line-height      : 40px;
   background-color : #ffffff;
-  border-radius    : 6px;
+  text-align       : center;
+  font-size        : 16px;
+  box-shadow       : 2px 2px 2px #e8e9eb;
 }
 
-.focus:hover
+.reset-selections
 {
-  background-color : #dedede;
+  margin           : 10px;
+  text-align       : center;
+  width            : 90px;
+  height           : 20px;
+  cursor           : pointer;
```

Comment:
This is not necessary, because this is automatically applied for `button` elements (is part of the built-in browser styles).